test.sh.in

```diff
@@ -22,8 +24,12 @@ if [ ! -z "${2}" ]; then
 fi
 
 dlHttpFiles() {
-  dir=${WORKSPACE}/build/static/http/common
-  if [ "${REPO_NAME}" != "on-http" ]; then
+  if [ -n "${MULTI}" ]; then
+    dir=${WORKSPACE}/on-http/static/http/common
+  else
+    dir=${WORKSPACE}/build/static/http/common
+  fi
+  if [[ "${REPO_NAME}" != *"on-http"* ]]; then
```

Comment:
if [ -n "${MULTI}" ], the repos in REPO_NAME will be copied from their workspace to on-multi workspace.